app/Template/project_list/show.php

```diff
@@ -1,12 +1,14 @@
 <section id="main">
     <div class="page-header">
         <ul>
+            <?= $this->hook->render('template:projectlist:menubefore', array('project' => $project)) ?>
```

Comment:
You can send these changes in a separate PR to be merged quickly, do not forget to update the documentation if you add new hooks.